jobsdb/backup.go

```diff
@@ -553,17 +553,16 @@ func (jd *HandleT) createTableDumps(queryFunc func(int64) string, pathFunc func(
 			}
 			preferences, err := jd.fileUploaderProvider.GetStoragePreferences(workspaceID)
```

Comment:
I had to go through the actual implementation for understanding that the current implementation only throughs an error when there is no workspace present in settings. Using a sentinel error might be preferable here.